[INFO ] Logging to: /rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/outdir_train_configuration/RandomForest_Iris_scenario/log-run14.txt
[INFO ] Version of SMAC is v2.10.03-master-778 (3ee628ef9bf2), running on OpenJDK 64-Bit Server VM (1.8.0_442) and Linux 4.18.0-553.36.1.el8_10.x86_64 (amd64) 
[INFO ] Call String: smac --scenario-file Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/RandomForest_Iris_scenario.txt --seed 14 
[INFO ] SMAC started at: Apr 11, 2025 1:27:18 PM. Minimizing mean quality.
[INFO ] First incumbent: config 1 (internal ID: 0x000B), with mean quality: -1.0; estimate based on 1 runs.
[INFO ] Sample call for new incumbent config 1 (internal ID: 0x000B): 
cd /rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example; /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py Solvers/RandomForest Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp accuracy:max Instances/Iris/Iris.csv 0 60.0 2147483647 2702708 -bootstrap 'True' -criterion 'gini' -max_depth_type 'None' -max_features_special 'sqrt' -max_features_type 'special' -max_leaf_nodes_type 'None' -max_samples_type 'None' -min_impurity_decrease '0.0' -min_samples_leaf '1' -min_samples_split '2' -min_weight_fraction_leaf '0.0' -n_estimators '100' -oob_score 'True'  
[WARN ] [PROCESS-ERR]  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/tools/runsolver.py:184: RuntimeWarning: Solver output decoding failed from RunSolver configuration: '/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest/runsolver --timestamp --use-pty --cpu-limit 60.0 -w Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-11-14:27:38_75280_2968332943.log -v Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-11-14:27:38_75280_2968332943.val -o Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-11-14:27:38_75280_2968332943.rawres Solvers/RandomForest/sparkle_solver_wrapper.py '{"bootstrap": "False", "criterion": "log_loss", "max_depth": "93", "max_depth_type": "int", "max_features_float": "0.38804584534471276", "max_features_type": "float", "max_leaf_nodes": "511", "max_leaf_nodes_type": "int", "min_impurity_decrease": "0.4163999543913039", "min_samples_leaf": "20", "min_samples_split": "49", "min_weight_fraction_leaf": "0.04068257893278898", "n_estimators": "361", "solver_dir": "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest", "instance": "Instances/Iris/Iris.csv", "seed": "595167", "objectives": "accuracy:max", "cutoff_time": "60.0"}''. Setting status to 'UNKNOWN'.
[WARN ] [PROCESS-ERR]    warnings.warn("Solver output decoding failed from RunSolver configuration: "
[WARN ] [PROCESS-ERR]  Traceback (most recent call last):
[WARN ] [PROCESS-ERR]    File "/rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py", line 46, in <module>
[WARN ] [PROCESS-ERR]      quality = float(output[objective.name])
[WARN ] [PROCESS-ERR]  TypeError: float() argument must be a string or a real number, not 'NoneType'
[ERROR] The following algorithm call failed: cd "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example" ;  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py Solvers/RandomForest Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp accuracy:max Instances/Iris/Iris.csv 0 60.0 2147483647 595167 -bootstrap 'False' -criterion 'log_loss' -max_depth '93' -max_depth_type 'int' -max_features_float '0.38804584534471276' -max_features_type 'float' -max_leaf_nodes '511' -max_leaf_nodes_type 'int' -min_impurity_decrease '0.4163999543913039' -min_samples_leaf '20' -min_samples_split '49' -min_weight_fraction_leaf '0.04068257893278898' -n_estimators '361'  
[WARN ] Detected CRASHED run. The Solution Quality of CRASHED runs will be transformed to the MAX(quality, 1.0E9), to disable this use --transform-crashed-quality false to change the value use --transform-crashed-quality-value
[WARN ] [PROCESS-ERR]  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/tools/runsolver.py:184: RuntimeWarning: Solver output decoding failed from RunSolver configuration: '/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest/runsolver --timestamp --use-pty --cpu-limit 60.0 -w Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-11-14:27:53_75793_3886959222.log -v Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-11-14:27:53_75793_3886959222.val -o Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-11-14:27:53_75793_3886959222.rawres Solvers/RandomForest/sparkle_solver_wrapper.py '{"bootstrap": "True", "criterion": "entropy", "max_depth_type": "None", "max_features_float": "0.6715473115033171", "max_features_type": "float", "max_leaf_nodes": "824", "max_leaf_nodes_type": "int", "max_samples_type": "None", "min_impurity_decrease": "0.0740537713374203", "min_samples_leaf": "32", "min_samples_split": "30", "min_weight_fraction_leaf": "0.07042025824240783", "n_estimators": "278", "oob_score": "False", "solver_dir": "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest", "instance": "Instances/Iris/Iris.csv", "seed": "2702708", "objectives": "accuracy:max", "cutoff_time": "60.0"}''. Setting status to 'UNKNOWN'.
[WARN ] [PROCESS-ERR]    warnings.warn("Solver output decoding failed from RunSolver configuration: "
[WARN ] [PROCESS-ERR]  Traceback (most recent call last):
[WARN ] [PROCESS-ERR]    File "/rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py", line 46, in <module>
[WARN ] [PROCESS-ERR]      quality = float(output[objective.name])
[WARN ] [PROCESS-ERR]  TypeError: float() argument must be a string or a real number, not 'NoneType'
[ERROR] The following algorithm call failed: cd "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example" ;  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py Solvers/RandomForest Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp accuracy:max Instances/Iris/Iris.csv 0 60.0 2147483647 2702708 -bootstrap 'True' -criterion 'entropy' -max_depth_type 'None' -max_features_float '0.6715473115033171' -max_features_type 'float' -max_leaf_nodes '824' -max_leaf_nodes_type 'int' -max_samples_type 'None' -min_impurity_decrease '0.0740537713374203' -min_samples_leaf '32' -min_samples_split '30' -min_weight_fraction_leaf '0.07042025824240783' -n_estimators '278' -oob_score 'False'  
[WARN ] [PROCESS-ERR]  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/tools/runsolver.py:184: RuntimeWarning: Solver output decoding failed from RunSolver configuration: '/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest/runsolver --timestamp --use-pty --cpu-limit 60.0 -w Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-11-14:28:10_76347_937043327.log -v Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-11-14:28:10_76347_937043327.val -o Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-11-14:28:10_76347_937043327.rawres Solvers/RandomForest/sparkle_solver_wrapper.py '{"bootstrap": "False", "criterion": "gini", "max_depth": "48", "max_depth_type": "int", "max_features_special": "sqrt", "max_features_type": "special", "max_leaf_nodes_type": "None", "min_impurity_decrease": "0.13324386392490534", "min_samples_leaf": "37", "min_samples_split": "97", "min_weight_fraction_leaf": "0.46460990411369013", "n_estimators": "453", "solver_dir": "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest", "instance": "Instances/Iris/Iris.csv", "seed": "4156112", "objectives": "accuracy:max", "cutoff_time": "60.0"}''. Setting status to 'UNKNOWN'.
[WARN ] [PROCESS-ERR]    warnings.warn("Solver output decoding failed from RunSolver configuration: "
[WARN ] [PROCESS-ERR]  Traceback (most recent call last):
[WARN ] [PROCESS-ERR]    File "/rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py", line 46, in <module>
[WARN ] [PROCESS-ERR]      quality = float(output[objective.name])
[WARN ] [PROCESS-ERR]  TypeError: float() argument must be a string or a real number, not 'NoneType'
[ERROR] The following algorithm call failed: cd "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example" ;  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py Solvers/RandomForest Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp accuracy:max Instances/Iris/Iris.csv 0 60.0 2147483647 4156112 -bootstrap 'False' -criterion 'gini' -max_depth '48' -max_depth_type 'int' -max_features_special 'sqrt' -max_features_type 'special' -max_leaf_nodes_type 'None' -min_impurity_decrease '0.13324386392490534' -min_samples_leaf '37' -min_samples_split '97' -min_weight_fraction_leaf '0.46460990411369013' -n_estimators '453'  
[WARN ] [PROCESS-ERR]  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/tools/runsolver.py:184: RuntimeWarning: Solver output decoding failed from RunSolver configuration: '/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest/runsolver --timestamp --use-pty --cpu-limit 60.0 -w Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-11-14:28:42_77268_3457010022.log -v Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-11-14:28:42_77268_3457010022.val -o Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-11-14:28:42_77268_3457010022.rawres Solvers/RandomForest/sparkle_solver_wrapper.py '{"bootstrap": "True", "criterion": "log_loss", "max_depth": "11", "max_depth_type": "int", "max_features_float": "0.9221887246754885", "max_features_type": "float", "max_leaf_nodes": "191", "max_leaf_nodes_type": "int", "max_samples_type": "None", "min_impurity_decrease": "0.4689569534931672", "min_samples_leaf": "13", "min_samples_split": "99", "min_weight_fraction_leaf": "0.09327714398919962", "n_estimators": "139", "oob_score": "False", "solver_dir": "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest", "instance": "Instances/Iris/Iris.csv", "seed": "2702708", "objectives": "accuracy:max", "cutoff_time": "60.0"}''. Setting status to 'UNKNOWN'.
[WARN ] [PROCESS-ERR]    warnings.warn("Solver output decoding failed from RunSolver configuration: "
[WARN ] [PROCESS-ERR]  Traceback (most recent call last):
[WARN ] [PROCESS-ERR]    File "/rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py", line 46, in <module>
[WARN ] [PROCESS-ERR]      quality = float(output[objective.name])
[WARN ] [PROCESS-ERR]  TypeError: float() argument must be a string or a real number, not 'NoneType'
[ERROR] The following algorithm call failed: cd "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example" ;  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py Solvers/RandomForest Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp accuracy:max Instances/Iris/Iris.csv 0 60.0 2147483647 2702708 -bootstrap 'True' -criterion 'log_loss' -max_depth '11' -max_depth_type 'int' -max_features_float '0.9221887246754885' -max_features_type 'float' -max_leaf_nodes '191' -max_leaf_nodes_type 'int' -max_samples_type 'None' -min_impurity_decrease '0.4689569534931672' -min_samples_leaf '13' -min_samples_split '99' -min_weight_fraction_leaf '0.09327714398919962' -n_estimators '139' -oob_score 'False'  
[WARN ] [PROCESS-ERR]  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/tools/runsolver.py:184: RuntimeWarning: Solver output decoding failed from RunSolver configuration: '/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest/runsolver --timestamp --use-pty --cpu-limit 60.0 -w Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-11-14:28:58_77632_615516089.log -v Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-11-14:28:58_77632_615516089.val -o Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-11-14:28:58_77632_615516089.rawres Solvers/RandomForest/sparkle_solver_wrapper.py '{"bootstrap": "False", "criterion": "log_loss", "max_depth_type": "None", "max_features_float": "0.5117889236330008", "max_features_type": "float", "max_leaf_nodes": "430", "max_leaf_nodes_type": "int", "min_impurity_decrease": "0.22316560053664053", "min_samples_leaf": "58", "min_samples_split": "84", "min_weight_fraction_leaf": "0.4217470851434753", "n_estimators": "462", "solver_dir": "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest", "instance": "Instances/Iris/Iris.csv", "seed": "4156112", "objectives": "accuracy:max", "cutoff_time": "60.0"}''. Setting status to 'UNKNOWN'.
[WARN ] [PROCESS-ERR]    warnings.warn("Solver output decoding failed from RunSolver configuration: "
[WARN ] [PROCESS-ERR]  Traceback (most recent call last):
[WARN ] [PROCESS-ERR]    File "/rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py", line 46, in <module>
[WARN ] [PROCESS-ERR]      quality = float(output[objective.name])
[WARN ] [PROCESS-ERR]  TypeError: float() argument must be a string or a real number, not 'NoneType'
[ERROR] The following algorithm call failed: cd "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example" ;  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py Solvers/RandomForest Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp accuracy:max Instances/Iris/Iris.csv 0 60.0 2147483647 4156112 -bootstrap 'False' -criterion 'log_loss' -max_depth_type 'None' -max_features_float '0.5117889236330008' -max_features_type 'float' -max_leaf_nodes '430' -max_leaf_nodes_type 'int' -min_impurity_decrease '0.22316560053664053' -min_samples_leaf '58' -min_samples_split '84' -min_weight_fraction_leaf '0.4217470851434753' -n_estimators '462'  
